mozilla/blurts-server
# MNTOR-981: Fxa Events

# References: 
Jira: MNTOR-981




# Description
Set up FxA event callback URI endpoint to receive and handle events from FxA. Some events we can expect are:
* User password changes
* User profile change (pending FxA investigation, currently not getting any profile change events)
* User deletion
* User subscription changes

This PR contains the logic for handling the first 3. We will wait for the SubPlat work to be done before handling the subscription changes


# How to test
To test the code, we need to take over heroku server, and push this change there. The testing needs to be done on dev because of the URI where the callback is pointing to.
* watch the logs of the heroku server
* create a new account in monitor
* change the FxA password in FxA settings
* receive a `password-change` events, see it in the logs
* delete the account
* receive a `delete` event
* check the database row to make sure that the user is deleted from both `subscriber` and `email_address` tables

# Screenshots

<img width="1403" alt="Screenshot 2023-05-04 at 1 01 27 PM" src="https://user-images.githubusercontent.com/42494162/236519283-577a525b-6d60-49f2-beee-ad96ac53258b.png">

<img width="1405" alt="Screenshot 2023-05-04 at 1 01 39 PM" src="https://user-images.githubusercontent.com/42494162/236519243-a9d246d4-0455-4692-b652-1e58291b6d0d.png">


# Checklist (Definition of Done)
